misc/python/materialize/cli/ci_logged_errors_detect.py

```diff
@@ -56,7 +56,7 @@
     # Expected once compute cluster has panicked, only happens in CI
     (?!.*aborting\ because\ propagate_crashes\ is\ enabled)
     # Emitted by webhook source tests that explicitly panic the validation.
-    (?!.*panic\ while\ validating\ webhook\ request')
+    (?!.*webhook\ panic\ test')
```

Comment:
note: this isn't relevant to IN CLUSTER change, but is a small improvement that was just easy to include here